emsdk/tests/test_emulate.py

```diff
@@ -21,6 +21,8 @@ def test_emulate_function(tmpdir):
         subprocess.run(
             [
                 "emcc",
+                "-s"
+                'BINARYEN_EXTRA_PASSES="--pass-arg=max-func-params@61"',
```

Comment:
Why are these necessary for the test to pass? I think it should do fine without it

Reply:
Yeah, just left in from earlier testing. Will remove.